environment/qa4sm_env.yml

```diff
@@ -166,7 +166,7 @@ dependencies:
     - pytest-cov==2.10.1
     - pytest-django==4.1.0
     - pytest-mpl==0.12
-    - qa4sm-reader==0.3.3
+    - git+https://github.com/TUW-GEO/qa4sm-reader.git 
```

Comment:
might need to define a @branch at the end

Reply:
As in: git+https://github.com/TUW-GEO/qa4sm-reader.git@master ?

Reply:
Oh, great, but still we have to avoid it while doing the deployment :D

Reply:
In the meanwhile I updated my local environment with just git+https://github.com/TUW-GEO/qa4sm-reader.git  and it worked

Reply:
ok then I assume it uses the default branch, makes sense

Reply:
yes @sheenaze as long as we run the script to generate a new env yml before deployment it should be fine ;)

Reply:
yeah, but if you don't do a new release before the deployment, the new env will refer to a wrong version.

Reply:
of course ;) just let me know when when everything works properly and I will trigger the new release